---
abstract: >-
  BAli-Phy, a popular Bayesian method that co-estimates multiple sequence
  alignments and phylogenetic trees, is a rigorous statistical method, but due
  to its computational requirements, it has generally been limited to relatively
  small datasets (at most about 100 sequences). Here, we repurpose BAli-Phy as a
  ‘phylogeny-aware’ alignment method: we estimate the phylogeny from the input
  of unaligned sequences, and then use that as a fixed tree within BAli-Phy.


  We show that this approach achieves high accuracy, greatly superior to Prank, the current most popular phylogeny-aware alignment method, and is even more accurate than MAFFT, one of the top performing alignment methods in common use. Furthermore, this approach can be used to align very large datasets (up to 1000 sequences in this study).
slides: ""
url_pdf: ""
publication_types:
  - "2"
authors:
  - Maya Gupta
  - Paul Zaharias
  - Tandy Warnow
author_notes:
  - First Author
  - Second Author
  - PI
publication: Bioinformatics
summary: ""
url_dataset: ""
url_project: ""
publication_short: ""
url_source: ""
url_video: ""
title: Accurate large-scale phylogeny-aware alignment using BAli-Phy
doi: https://doi.org/10.1093/bioinformatics/btab555
featured: false
tags: []
projects: []
image:
  caption: ""
  focal_point: ""
  preview_only: false
date: 2021-07-28T04:00:00.000Z
url_slides: ""
publishDate: 2017-01-01T00:00:00Z
url_poster: ""
url_code: ""
---
